charms/jupyter-ui/tests/integration/test_charm.py

```diff
@@ -68,3 +69,41 @@ async def test_ui_is_accessible(ops_test: OpsTest):
     assert result_status == 200
     assert len(result_text) > 0
     assert "Jupyter Management UI" in result_text
+
+
+@pytest_asyncio.fixture
+async def deploy_kubeflow_dashboard(ops_test: OpsTest):
+    """Deploys kubeflow-dashboard and kubeflow-profiles, cleaning them up after the usage."""
+    kubeflow_profiles = "kubeflow-profiles"
+    await ops_test.model.deploy(kubeflow_profiles, channel="stable", trust=True)
+
+    kubeflow_dashboard = "kubeflow-dashboard"
+    # Requires latest/edge until https://github.com/canonical/kubeflow-dashboard-operator/pull/134
+    # is merged into latest/stable
+    ops_test.model.deploy(kubeflow_dashboard, channel="edge", trust=True)
```

Comment:
Why edge?

Reply:
discussed offline.  keeping edge because our stable for this charm isn't up to date yet